code/modules/projectiles/projectile.dm

```diff
@@ -67,10 +67,26 @@
 	qdel(src)
 
 /obj/item/projectile/proc/on_hit(atom/target, blocked = 0, hit_zone)
+	var/turf/target_loca = get_turf(target)
 	if(!isliving(target))
 		return 0
 	var/mob/living/L = target
 	if(blocked < 100) // not completely blocked
+		if(damage && L.blood_volume && damage_type == BRUTE)
+			var/splatter_dir = dir
+			if(starting)
+				splatter_dir = get_dir(starting, target_loca)

```

Comment:
@KasparoVy Woah, neat!

Though I don't think you need to re-clone to make a PR on fox's branch, you should be able to just do that, since both repos are downstream of paradise